native/cocos/gi/light-probe/Delaunay.cpp

```diff
@@ -62,28 +62,21 @@ Tetrahedron::Tetrahedron(const Delaunay *delaunay, int32_t v0, int32_t v1, int32
     }
 }
 
-void Delaunay::build(const ccstd::vector<Vec3> &points) {
-    reset();
-
-    const auto pointCount = points.size();
-    if (pointCount < 4) {
-        debug::warnID(17000);
-        return;
-    }
-
-    _probes.reserve(pointCount);
-    for (const auto &point : points) {
-        _probes.emplace_back(point);
-    }
+ccstd::vector<Tetrahedron> Delaunay::build(const ccstd::vector<Vertex> &probes) {
```

Comment:
`probes` might be moved into`_probes`, if it is not used anywhere else.

Reply:
Yes, you are right, but I have no time to do careful optimization. Delaunay's implemtation in c++ is not used now.